universal-application-tool-0.0.1/app/views/admin/questions/QuestionsListView.java

```diff
@@ -180,12 +180,12 @@ private Tag renderQuestionTextCell(QuestionDefinition definition) {
     String questionHelpText = "";
 
     try {
-      questionText = definition.getQuestionText(LocalizationUtils.DEFAULT_LOCALE);
+      questionText = definition.getQuestionText().get(LocalizedStrings.DEFAULT_LOCALE);
```

Comment:
question: could we use `getDefault()` here? perhaps not, since we don't want it to throw

Reply:
Ah good point - I think `getDefault` should throw a `RuntimeException`, so this seems good to keep as-is